parser-sdk/nodejs/parser-utils.js

```diff
@@ -0,0 +1,41 @@
+const fs = require("fs");
+const Ajv = require("ajv-draft-04");
+const ajv = new Ajv();
+const util = require("util");
+// eslint-disable-next-line security/detect-non-literal-fs-filename
+const readFile = util.promisify(fs.readFile);
+const { v4: uuid } = require("uuid");
+
+function addIdsAndDates(findings) {
+  return findings.map((finding) => {
+    return {
+      ...finding,
+      id: uuid(),
+      parsed_at: new Date().toISOString(),
+    };
+  });
+}
+
+// eslint-disable-next-line
+async function validate(jsonData) {
```

Comment:
```suggestion
async function validateJsonSchema(jsonData) {
```

Reply:
What about validateWithJsonSchema? validateJsonSchema sounds to me like I am validating a json schema but I'm validating Json Data with a JSON Schema.

Reply:
Does `validateAgainstJsonSchema` sound better?

Reply:
it does! I changed it to `validateAgainstJsonSchema `